dspace-api/src/main/java/org/dspace/discovery/IndexableObject.java

```diff
@@ -21,7 +21,7 @@
  * @param <PK>
  *            the Class of the primary key
  */
-public interface IndexableObject<T extends ReloadableEntity, PK extends Serializable> {
+public interface IndexableObject<T extends ReloadableEntity<PK>, PK extends Serializable> {
```

Comment:
I'm unclear why this change is needed.  It's not obvious to me where this is being used in this PR.

Reply:
it is not a real change, I have only noted that the reference to the generic ReloadableEntity was not parameterized so I have added it to remove the warning